components/Card.js

```diff
@@ -21,13 +21,29 @@ export default function Card({
         </DeleteButton>
       )}
       <p>Date: {date}</p>
-      {weight && <p>Weight: {weight}</p>}
-      {height && <p>Height: {height}</p>}
-      {feastTime && (
-        <p>
-          Nurse time: {feastTime.substr(0, 2)}:{feastTime.substr(2, 2)}:
-          {feastTime.substr(4, 2)}
-        </p>
+      {weight?.[0] && (
+        <ul>
+          {weight.map((weight, index) => (
+            <li key={index}>Weight: {weight} </li>
+          ))}
+        </ul>
+      )}
+      {height?.[0] && (
+        <ul>
+          {height.map((height, index) => (
+            <li key={index}>Height: {height} </li>
+          ))}
+        </ul>
+      )}
+      {feastTime?.[0] && (
+        <ul>
+          {feastTime.map((entry, index) => (
+            <li key={index}>
```

Comment:
please don't use the index here, get an index from your database for that. (Tipp: to display you don't have to chop your datapoints into weight, height, etc... you can hand over the whole object with all information and then only use the parts that are insteresting for that specific list).

Reply:
I'm not sure I understand your feedback on this one. I thought I am already getting the index from that array, which comes from my db?